coordinator/src/main/scala/filodb.coordinator/queryplanner/ShardKeyRegexPlanner.scala

```diff
@@ -259,24 +258,23 @@ class ShardKeyRegexPlanner(val dataset: Dataset,
     //   Columns with higher cardinalities should be defined last.
     val shardKeyCols = dataset.options.shardKeyColumns.filterNot(_ == dataset.options.metricColumn)
     val nonMetricShardKeyColToIndex = shardKeyCols.zipWithIndex.toMap
-    val partitionToKeyGroups = partitionsToNonSplitKeys.map{ case (partition, keys) =>
+    val assignmentToKeyGroups = assignmentToNonSplitKeys.map{ case (assignment, keys) =>
       val prefixGroups = keys
         .map(key => key.sortBy(filter => nonMetricShardKeyColToIndex.get(filter.column)))
         .groupBy(_.dropRight(1))
         .values
         .flatMap(group => group.grouped(qContext.plannerParams.maxShardKeyRegexFanoutBatchSize))
-      (partition, prefixGroups)
+      (assignment, prefixGroups)
     }
 
     // Skip lower-level aggregate presentation if there is more than one plan to materialize.
     // In that case, the presentation step will be applied by this planner.
     val skipAggregatePresentValue = !pushdownPlan &&
-                                      partitionToKeyGroups.values.map(_.size).sum + partitionSplitKeys.size > 1
-
+      assignmentToKeyGroups.values.map(_.size).sum + assignmentSplitKeys.size > 1
     // Materialize a plan for each group of non-split of keys.
     // Each group will be encoded into a set of Equals/EqualsRegex filters, where regex filters
     //   contain only shard-key values concatenated by pipes.
-    val nonSplitPlans = partitionToKeyGroups.flatMap{ case (partition, keyGroups) =>
+    val nonSplitPlans = assignmentToKeyGroups.flatMap{ case (assignment, keyGroups) =>
```

Comment:
I appreciate all these renames of `partition` -> `assignment`. That is more accurate regardless of the other changes in this PR.